core/rawdb/freezer_table.go

```diff
@@ -46,21 +46,20 @@ var (
 	errNotSupported = errors.New("this operation is not supported")
 )
 
-// indexEntry contains the number/id of the file that the data resides in, aswell as the
-// offset within the file to the end of the data
+// indexEntry contains the number/id of the file that the data resides in, as well as the
```

Comment:
I think `aswell` is a proper actual word

Reply:
yup, i can change it back.